<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterPanel</class>
 <widget class="QWidget" name="FilterPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="partitionBox">
         <property name="title">
          <string>Partition</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="partComboBox"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Bins:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="partBinSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Min:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="partMinSpinBox"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Max:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="partMaxSpinBox"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="partitionList"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="flaggedBox">
         <property name="title">
          <string>Flagged Items</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>Ignore</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>Exclude From Selection</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>Keep Only Flagged Items</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="flaggedList"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="filtersBox">
      <property name="title">
       <string>Filters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="saveFiltersButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadFiltersButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="filterTreeView">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
